META6.json

```diff
@@ -22,7 +22,7 @@
     ],
     "description" : "Perl 6 Kernel for Jupyter Notebooks",
     "test-depends" : [ ],
-    "version" : "0.0.11",
+    "version" : "0.0.12",
```

Comment:
Please don't bump this yet -- I will bump it and also increment the value returned by the server, as well as tag it in github.  A version bump will be its own commit.  (which will correspond to a tarball in case we start using cpan)

Reply:
I remove that squashing all commits
Sorry a bad design from a (usually) lonely worker.